tests/testenv/__init__.py

```diff
@@ -319,6 +321,11 @@ def start_elasticsearch(self):
         self.elasticsearch_process = ElasticSearchProcess()
         self.elasticsearch_process.start()
 
+    def start_postgresql(self):
```

Comment:
since systemd not exists in all system, don't use it. just tell the developer to make sure that he have running ps.....
no need for postgresql process object